.github/workflows/validate.yaml

```diff
@@ -112,7 +112,7 @@ jobs:
                 -level="error"
 
       - name: Go Lint - knative-operator
-        uses: golangci/golangci-lint-action@v2
+        uses: golangci/golangci-lint-action@v3
         with:
           version: v1.43
```

Comment:
@matzew @skonto @pierDipi should we stick with whatever upstream actions use or just go latest by default?

Reply:
I updated to latest #1465 ;)

Reply:
I have no real opinion as long as it works and is useful.